script.embuary.helper/service.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/python
+# coding: utf8
```

Comment:
This throws an exception with Python3, I haven't tested with Python 2.
```suggestion
# coding: utf-8
```

Reply:
Yeah? haven't got any at my end but I'll change it.